lib/components/LinearProgressBar/LinearProgressBar.scss

```
@@ -0,0 +1,108 @@
+$inactive_color: #999999;
+$success_color: #4bb4b3;
+$warning_color: #ffad3a;
+$error_color: #ec3f3f;
+$active_color:#d6dadb;
+$border_section:#f2f3f4;
+$state_hover:#e6e9e9;
+$BORDER_RADIUS:4px;
```

Let's use the CSS/SCSS variables from `root.scss` or `variables.scss`. 

For example: 
`var(--status-sucess, #4bb4b3) ` 
FYI: for now let's append the actual hex code of the color. So that whatever reason the css is not working properly we have a fallback value to rely on 😃 👍🏼

---

@aizad-deriv I tried to add values from the root.scss file, but the corresponding styles do not appear in the component during testing.

---

```suggestion
$inactive_color: var(--general-main-3, #999999);
$success_color: var(--status-success, #4bb4b3 );
$warning_color: var(--status-warning, #ffad3a);
$error_color: var(--status-danger, #ec3f3f);
$active_color:var(--state-active, #d6dadb);
$border_section:var(--border-divider, #f2f3f4);
$state_hover:var(--general-hover, #e6e9e9);
$BORDER_RADIUS:4px;
```
please change them to these